content/contact.hbs

```diff
@@ -15,9 +15,15 @@
 	<header class="section">
 		<div class="container">
 			<h1>Join in <span class="semester-text">Spring 2021</span> or later</h1>
-			<p class="has-text-centered">To enroll for a future semester, fill out our interest form and we'll contact you with more details including when and how to apply.</p>
+			<p class="has-text-centered">If you are a college or highschool student, to enroll for a future semester, fill out our interest form and we'll contact you with more details including when and how to apply.</p>
 			<p class="has-text-centered"><a class="button is-large future-interest" href="https://docs.google.com/forms/d/e/1FAIpQLSfkvC-q9kdtHttOnJVYtJ4vA26Jpkqbdz1m1LS0h-perXGnhQ/viewform?usp=sf_link">Future Cadet Interest Form</a></p>
 		</div>
+		<div class="container">

```

Comment:
Move into the above `<div class="container">` to avoid breaking the page styling because there should only be a single container